tests/scripts/multus-test.sh

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env bash
+set -eEuo pipefail
+
+##
+## This script will install a configurable number of multus-networked daemonsets to test that your
+## multus configuration and network hardware can provide enough IP addresses for Ceph.
+## Configuration options are commented below.
+##
+## TODO: in the future, this test should also check network stability under a reasonable amount of
+##       load to/from daemons.
+##
+
+## A comma-delimited list of multus networks to use for test pods
+: "${MULTUS_NETWORKS:="public-net,cluster-net"}"
+
+## The number of daemons to run per node as a test. This should reflect the overall size expected
+## for your Ceph install. As an easy guideline, add a daemon for each additional OSD beyond 3.
```

Comment:
I see that this is used for the number of daemonsets to be created, but not sure I understand if it's tied to the OSD or node count. Is this a correct interpretation for these two examples?
1. If they have 4 OSDs across 4 nodes, it should be set to 1? 
2. If they have 9 OSDs across 3 nodes, it should be set to 6?

Reply:
You are correct in part. It should not be set to 1 or 6, but 1/6 should be added to the default value.